.github/workflows/on-pull-request.yml

```diff
@@ -42,5 +42,8 @@ jobs:
       - name: Install deps
         run: go mod download
 
+      - name: Lint
+        run: make lint
```

Comment:
Instead of referencing `make`, you can add something like:

```yaml
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v3
        with:
          version: ${{ env.GOLANGCI_LINT_VERSION }}
          skip-cache: true
```

And be sure to set that env var at top before the `jobs` stanza:

```yaml
env:
  GOLANGCI_LINT_VERSION: v1.54.2
```